_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -12,13 +12,11 @@ Some peoples' eyes can't tell the difference between certain colours; that's not
 
 Yet, colour blindness is not rare. Some form of colour blindness affects at least one in twenty people; [Wikipedia has a helpful table](https://en.wikipedia.org/wiki/Color_blindness#Epidemiology), based on [a journal article](https://apfmj.biomedcentral.com/articles/10.1186/s12930-014-0010-3). There are [several different forms](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/). The most common, red-green colourblindness, affects around one in twelve men.
 
-Colour blindness is less profound than many impairments. Yet, people with colour vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are colour blind from driving. Yet, the USA doesn't consider colour blindness as a disability, for workplace discrimination purposes.
-
-I've avoided going into the different types here, as the principles are the same for each. Also, [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness) has some excellent descriptions.
+Colour blindness is less debilitating than many impairments. Yet, people with colour vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are colour blind from driving. Yet, the USA doesn't consider colour blindness as a disability, for workplace discrimination purposes.rely've avoided going into the different types here, as the principles are the same for each. Also, [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness) has some excellent descriptions.
 
 ## What does this mean for accessibility on the web?
 
-Don't use on colour alone to communicate. As a result, you don't risk making life difficult for colour blind users. 
+Don't rely on colour alone to communicate. As a result, you can make life simpler for colour blind users. An example of this would be hyperlinks being designed to be underlined by default.
```

Comment:
Thanks for the first one, that should definitely be fixed. 

Regarding the second one (the example), This isn't about making life simple. It's about not making tasks difficult (or impossible), and I don't think they are opposites.

Reply:
no problem :) I get where you're coming from. As a colour blind (not red-green) person, I've definitely had experience of stuff being made unnecessarily difficult.